src/features/content-explorer/prop-types.js

```diff
@@ -43,7 +43,15 @@ const ItemsPropType = PropTypes.arrayOf(ItemOrPlaceholderPropType);
 
 const ItemsMapPropType = PropTypes.objectOf(ItemPropType);
 
+const BreadcrumbPropType = PropTypes.shape({
```

Comment:
Does it make sense to use the type from the `Breadcrumb` component instead of creating this new type?
https://github.com/box/box-ui-elements/blob/6c1566e82a3083d46aa6939307d29dd6daf7c3f5/src/components/breadcrumb/Breadcrumb.js#L14-L27

Reply:
React types and TS types are different and not interchangeable from what I understand